nasa-gibs/worldview
# WV-2536: Configuring Granule Layers

## Description

This adds granule layers to our config files allowing for the layers to be used in WV. 

## How To Test

1. `git checkout configuring-granule-layers`
2. `npm ci`
3. `npm run build`
4. `npm run watch`
5. [This link should load all 6 granule layers that are configured.](http://localhost:3000/?v=-224.8261200875947,-91.91884799362859,197.56024354876894,107.20615200637141&z=4&ics=true&ici=5&icd=10&l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,VIIRS_NOAA20_CorrectedReflectance_BandsM3-I3-M11_Granule(count=10),VIIRS_NOAA20_CorrectedReflectance_BandsM11-I2-I1_Granule(count=10),VIIRS_NOAA20_CorrectedReflectance_TrueColor_Granule(count=10),VIIRS_SNPP_CorrectedReflectance_BandsM3-I3-M11_Granule(count=10),VIIRS_SNPP_CorrectedReflectance_BandsM11-I2-I1_Granule(count=10),VIIRS_SNPP_CorrectedReflectance_TrueColor_Granule(count=10),VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor(hidden)&lg=false&t=2022-12-14-T21%3A47%3A38Z)
6. Show and hide each granule layer one by one and verify that you can view its associated imagery. (Granule layers will not have any imagery prior to December 13th).
